js/storage.js

```diff
@@ -27,6 +27,7 @@
 
     forkTheme: function(theme, title) {
       delete theme.id;
+      delete theme.manifestURL;
```

Comment:
otherwise duplicating an "enabled" theme will create a theme that you can't enable.